runtime/hydradx/src/lib.rs

```diff
@@ -581,21 +574,21 @@ impl orml_currencies::Config for Runtime {
 	type WeightInfo = weights::currencies::HydraWeight<Runtime>;
 }
 
-pub struct GalacticCouncilOrVestingOrRoot;
+pub struct VestingOrRoot;
 
-impl EnsureOrigin<Origin> for GalacticCouncilOrVestingOrRoot {
+impl EnsureOrigin<Origin> for VestingOrRoot {
 	type Success = AccountId;
 
 	fn try_origin(o: Origin) -> Result<Self::Success, Origin> {
 		Into::<Result<RawOrigin<AccountId>, Origin>>::into(o).and_then(|o| match o {
 			RawOrigin::Signed(caller) => {
```

Comment:
isn't this branch completely redundant as Vesting account shouldn't be able to sign anything and we dont have sudo to `sudoAs` it